src/hooks/useMint.ts

```diff
@@ -1,18 +1,43 @@
 import { useStore } from 'src/store';
 import { computed } from 'vue';
 import { Contract } from 'web3-eth-contract';
+import { stringToHex } from '@polkadot/util';
+import { getInjector } from 'src/modules/wallet/utils';
+import { decodeAddress } from '@polkadot/util-crypto';
+import { u8aToHex } from '@polkadot/util';
+
+const signMessage = stringToHex('Sign this to register to AstarBase for:');
+const PREFIX = '3c42797465733e';
+const POSTFIX = '3c2f42797465733e';
 
 export const useMint = () => {
   const store = useStore();
   const mintContract = computed(() => store.getters['general/mintContract']);
   const mintContractAddress = computed(() => store.getters['general/mintContractAddress']);
   const account = computed(() => store.getters['general/ethereumAccount']);
+  const substrateAccounts = computed(() => store.getters['general/substrateAccounts']);
+  const substrateAccount = computed(() => store.getters['general/substrateAccount']);
 
   // Todo: Modify the logic once contract has been finalized
-  const mint = () => {
+  const mint = async () => {
```

Comment:
rename `mint` everywhere. It is not minting function

Reply:
Done